Library/Homebrew/dev-cmd/merge-homebrew.rb

```diff
@@ -18,7 +23,7 @@ def git_merge
     puts "End   commit: #{end_sha1}"
 
     safe_system "git", "checkout", "master"
-    safe_system "git", "merge", "homebrew/master", "-m", "Merge branch homebrew/master into linuxbrew/master"
+    safe_system "git", "merge", "--no-ff", "homebrew/master", "-m", '"Merge branch homebrew/master into linuxbrew/master"'
```

Comment:
What situation brought up the need to add `--no-ff`? Do you have your `~/.gitconfig` set up to fast-forward merge?

`'"Merge branch homebrew/master into linuxbrew/master"'`
This doesn't look right. Why both single and double quotes?